start.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-npm install;
-npm install -g bower
+sudo npm install
```

Comment:
Bower is not supposed to be run as a super-user (their specifications), and since this script invokes bower, I figured that we should just run the script with as little permission as possible while still functioning.